drivers/net/dsa/lantiq_gsw_platform.c

```diff
@@ -32,22 +33,36 @@
 #include "lantiq_gsw.h"
 #include "lantiq_pce.h"
 
-static u32 gsw_platform_read(struct gswip_priv *priv, void *address)
+struct gsw_platform {
+	struct platform_device *platform_dev;
+	struct gswip_priv common;
+};
+
+static u32 gsw_platform_read(struct gswip_priv *priv, void *addr)
+{
+	(void*)priv;
```

Comment:
Compiler still seemed to output unused variable warnings, though not if no other errors existed? Not sure exactly what compiler settings my Yocto environment is using.
Took a quick look around and didn't see an existing macro to declare something unused but would use here if it existed.

Reply:
You might need to remove the `*` from `(void*)`